azure-pipelines.yml

```diff
@@ -79,14 +61,16 @@ jobs:
       versionSpec: '$(python.version)'
       architecture: 'x64'
 
-  - script: python -m pip install --upgrade pip
+  - script: |
+      python -m pip install --upgrade pip
+      python -m pip install -r requirements.txt
     displayName: 'Install dependencies'
 
   - script: |
       python -c "import locale; print('Preferred encoding:', locale.getpreferredencoding())"
       python -c "import sys; print('stdout encoding:', sys.stdout.encoding)"
       env
-      pip install pytest
+      python -m mypy wasabi
```

Comment:
Split this into two separate steps so a mypy failure leads to a clear CI failure. (I'm pretty sure it uses the exit code of the final command in the step.)

Reply:
Updated in https://github.com/explosion/wasabi/pull/22/commits/590c00847b239bf59a142571f8bc1f7354097b20